cmd/rpc/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 	cmd.Addr = addr
 	cmd.Method, _ = args["<method>"].(string)
 	cmd.Args, _ = args["<args>"].([]string)
+	cmd.UserAgent = fmt.Sprintf("Segment (rpc-cli/%s", version)
```

Comment:
This would conflict with https://github.com/segmentio/rpc/blob/master/cmd/jsonrpc/main.go#L37. Do you think we should try to rename it here instead?

Reply:
lol curses! How does `rpc-cli-legacy` sound to you?